scripts/expand-env.go

```diff
@@ -0,0 +1,41 @@
+// Copyright (c) 2019, Sylabs Inc. All rights reserved.
+// This software is licensed under a 3-clause BSD license. Please consult the
+// LICENSE.md file distributed with the sources of this project regarding your
+// rights to use or distribute this software.
+
+// expand-env is a simple program that reads stdin, replaces all
+// occurences of @VAR@ by the corresponding value of VAR in the current
+// environment, and writes the result to stdout.
+package main
```

Comment:
It it worth putting a newline right before the `package ...`

```suggestion

package main
```

Reply:
Thanks for looking at this.

> It it worth putting a newline right before the `package ...`

If you add a blank before `package` that becomes a comment. If you omit the blank, that's the package's documentation. It does not make a difference in this case because this piece of documentation cannot be accessed with the usual tools, but it's nice to follow conventions anyways.

Reply:
Good to know, okay :+1: